modules/tensor_mechanics/include/materials/ComputeAxisymmetricRZIncrementalStrain.h

```diff
@@ -0,0 +1,37 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef COMPUTEAXISYMMETRICRZINCREMENTALSTRAIN_H
+#define COMPUTEAXISYMMETRICRZINCREMENTALSTRAIN_H
+
+#include "Compute2DIncrementalStrain.h"
+
+/**
+ * ComputeAxisymmetricRZIncrementalStrain defines a strain increment only
+ * for incremental strains in an Axisymmetric simulation.
+ * The COORD_TYPE in the Problem block must be set to RZ.
+ */
+class ComputeAxisymmetricRZIncrementalStrain : public Compute2DIncrementalStrain
+{
+public:
+  ComputeAxisymmetricRZIncrementalStrain(const InputParameters & parameters);
+
+protected:
+  virtual void initialSetup();
+
+  /// Computes the current deformation gradient for axisymmetric problems, where
+  /// \epsilon_{\theta} = \frac{u_r}{r}
```

Comment:
if you enclose the equation in `\f$ ... \f$` it will get rendered correctly on the Doxygen pages!